protobufs/livekit_rtc_v2.proto

```diff
@@ -91,9 +132,23 @@ message ConnectResponse {
   repeated Codec enabled_publish_codecs = 8;
   bool fast_publish = 9;
   SessionDescription subscriber_sdp = 10;
+  SessionDescription publisher_sdp = 11;
 }
 
-message ErrorResponse {
-  uint32 message_id = 1;
-  string error = 2;
+enum AddTrackState {
+  UNUSED = 0;
+  ADDED = 1;
+  QUEUED = 2;
+}
+
+message RPC_AddAudioTrackResponse {
+  RPC_AddAudioTrackRequest rpc_add_audio_track_request = 1; // request reflected back
+  AddTrackState add_track_state = 2;
+  TrackInfo track_info = 3;
+}
+
+message RPC_AddVideoTrackResponse {
+  RPC_AddVideoTrackRequest rpc_add_video_track_request = 1; // request reflected back
+  AddTrackState add_track_state = 2;
+  TrackInfo track_info = 3;
```

Comment:
Although on the request side, splitting track type, keeping `TrackInfo` in the updates. But, an option is to make `AudioTrackInfo` and `VideoTrackInfo`, but that will be a massive change on server side if we were to keep two different types as a lot of code relies on `TrackInfo`.

However to ease the implementation effort, can keep `TrackInfo` internally and convert to `AudioTrackInfo` and `VideoTrackInfo` when sending updates. If having separate types would really help, can dig into it more.